__tests__/app.test.js

```diff
@@ -182,3 +181,51 @@ describe("GET: /api/articles/:article_id/comments", () => {
       });
   });
 });
+describe("POST: /api/articles/:article_id/comments", () => {
```

Comment:
Nice tests so far! A few more cases worth thinking about that it looks like you've missed thus far:

 Status 400: Bad request, invalid article_id
    -Responds with error object including status and message.
    E.g. POST /api/articles/notanumber/comments

Status 404: Not found, username does not exist 
( this means a valid username doesnt exist on the db, e.g. username: "godzilla123")
    -Responds with error object including status and message.